src/api/.rubocop.yml

```diff
@@ -32,6 +32,10 @@ Style/AndOr:
 Style/AsciiComments:
   Enabled: true
 
+# This cop checks how the *when*s of a case expression are indented in relation to its `case` or `end` keyword.
```

Comment:
How about "This cop checks that 'when' statements are indented on the same level like the 'case' they belong to"?